### PR TITLE
Clean up loose ends for new Baklava network client

### PR DESCRIPTION
**Pull request by nategraf**
### Description

Getting ready for the new Baklava launch I noticed a number of small but important loose ends that need to be accounted for. This PR includes:

* Change default gateway fee to 0 so that it is default off
* Change gateway fee processing to accept any recipient if the minimum fee is zero
* Change the client version to 0.9.0 to mark the first pre-release Celo version
* Prevent log spam from the reverting `getAddressFor` method before contracts are deployed
* Add a INFO or WARN log message when a static or trusted peer disconnects to catch cases where a proxy or validator drops their connection.
* Add a buffer to Istanbul chain head subscription to prevent possibly blocking the chain
* Refactor ethstats listener to ensure it never blocks subscriptions

### Tested

Existing unit tests and e2e tests as well as manual examination of the logs

### Backwards compatibility

100%

### CODE REVIEW

File: ethstats/ethstats.go [hunk 1]
```diff
@@ -233,15 +236,29 @@ func (s *Service) loop() {
...
+
+	go func() {
+		for {
+			select {
 			case delegateSignMsg := <-istDelegateSignCh:
 				var statsPayload StatsPayload
 				err := json.Unmarshal(delegateSignMsg.Payload, &statsPayload)
 				if err != nil {
-					break HandleLoop
+					log.Error("Error in unmarshalling selegate sign payload", "err", err, "payload", delegateSignMsg.Payload)
```
Comment (timmoreton) [inline ethstats/ethstats.go]: s/selegate/delegate

File: params/version.go [hunk 2]
```diff
@@ -21,10 +21,10 @@ import (
...
 
 const (
-	VersionMajor = 1        // Major version component of the current release
-	VersionMinor = 9        // Minor version component of the current release
-	VersionPatch = 8        // Patch version component of the current release
-	VersionMeta  = "stable" // Version metadata to append to the version string
+	VersionMajor = 0  // Major version component of the current release
+	VersionMinor = 9  // Minor version component of the current release
+	VersionPatch = 0  // Patch version component of the current release
+	VersionMeta  = "" // Version metadata to append to the version string
```
Comment (timmoreton) [inline params/version.go]: geth alternates this between `unstable` (not a released version) and `stable` (a released build whose commit corresponds to a tag) -- this would be nicer because otherwise I can't necessarily know this isn't *the* 1.9.0 build

File: ethstats/ethstats.go [hunk 3]
```diff
@@ -251,16 +268,15 @@ func (s *Service) loop() {
...
-				// TODO: This is a hacky measure to avoid blocking here if the
-				// channel is not consuming
+				// Avoid blocking here if the channel is not consuming
 				select {
 				case channel <- &statsPayload:
 				default:
 				}
-
+			case <-quitCh:
+				return
```
Comment (timmoreton) [inline ethstats/ethstats.go]: I think this works fine but the indentation is hard to follow - have you tested that it quits cleanly?

File: params/version.go (hunk 2 above)
Comment (nategraf) [reply params/version.go]: Got it, so right now it should be "unstable"

File: ethstats/ethstats.go (hunk 3 above)
Comment (nategraf) [reply ethstats/ethstats.go]: I haven't tested this yet (I wanted to get the reviews going), but I can do that!

Comment (nategraf) [reply ethstats/ethstats.go]: Decided to remove this 👍
